data/templates/router-advert/radvd.conf.tmpl

```diff
@@ -30,6 +30,17 @@ interface {{ iface }} {
     AdvOtherConfigFlag {{ 'on' if interface[iface].other_config_flag is defined else 'off' }};
     AdvRetransTimer {{ interface[iface].retrans_timer }};
     AdvCurHopLimit {{ interface[iface].hop_limit }};
+{%     for route in interface[iface].route %}
+    route {{ route }} {
+{%     if interface[iface].route[route].valid_lifetime is defined %}
```

Comment:
Please indent this by two more whitespaces as this is nested inside the `for` loop.